### PR TITLE
Enhance and fix translation on JSON api and AJAX

### PR DESCRIPTION
**Pull request by wudifeixue**
In this PR:

Fix Issue: #346 

Enhance and fix translation on challenge JSON api and AJAX.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json [hunk 1]
```diff
@@ -336,7 +336,7 @@
         "</div>"
       ],
       "tests": [
-        "assert(code.match(/filter/gi), 'message: You should be making use of the .filter method.');"
+        "assert(code.match(/filter/gi), 'message: 你应该使用 .filter 这个方法。');"
```
Comment (S1ngS1ng) [inline seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json]: Might wanna consider onsider `<code>.filter</code>`?

File: seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json [hunk 2]
```diff
@@ -345,16 +345,8 @@
...
-        "The navigator will get our user's current longitude and latitude.",
-        "You will see a prompt to allow or block this site from knowing your current location. The challenge can be completed either way, as long as the code is correct.",
-        "By selecting allow you will see the text on the output phone change to your latitude and longitude",
-        "Here's some code that does this:",
-        "<blockquote>if (navigator.geolocation) {</br>  navigator.geolocation.getCurrentPosition(function(position) {</br>    $(\"#data\").html(\"latitude: \" + position.coords.latitude + \"&lt;br&gt;longitude: \" + position.coords.longitude);</br>  });</br>}</blockquote>"
-      ],
-      "description": [
-        "我们还可以通过浏览器<code>navigator</code>获得我们当前所在的位置<code>geolocation</code>。",
-        "位置的信息包括经度<code>longitude</code>和纬度<code>latitude</code>。",
+        "在浏览器中还有一个很棒的功能就是我们还可以通过<code>navigator</code>来获得我们当前所在的地理位置<code>geolocation</code>。",
```
Comment (S1ngS1ng) [inline seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json]: 还有一件很棒的事情，我们可以获取用户的当前位置。每个浏览器都有自带的 <code>Navigator</code> 接口供我们调用。

**Review by S1ngS1ng — CHANGES_REQUESTED** [review]
Let me know in case of any further questions.

File: seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json (hunk 2 above)
Comment (wudifeixue) [reply seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json]: 昨晚我就想了有点怪，不过感觉怎么感觉还是很绕口233 不过再改改吧。

**Review by S1ngS1ng — APPROVED** [review]
LGTM :100:
